helpers/aws.ts

```diff
@@ -1,22 +1,25 @@
-const S3 = require('aws-sdk/clients/s3');
-
-// TODO: figure out where this should go
-const BUCKET = 'gitchaos';
```

Comment:
I've added this to a process.env - so remind me to send you the updated one for running locally